main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -7,10 +7,31 @@
     "java.lang.Runtime::totalMemory": "Env, IO",
     "java.lang.Runtime::maxMemory": "Env, IO",
     "java.lang.Runtime::version": "Env",
+    "java.lang.System::setIn": "Sys, IO",
+    "java.lang.System::setOut": "Sys, IO",
+    "java.lang.System::setErr": "Sys, IO",
     "java.lang.System::console": "IO",
+    "java.lang.System::inheritedChannel": "Sys, IO",
+    "java.lang.System::setSecurityManager": "Env, Sys, IO",
+    "java.lang.System::getSecurityManager": "Env, Sys, IO",
     "java.lang.System::currentTimeMillis": "IO",
-    "java.lang.System::getenv": "Env, IO",
     "java.lang.System::nanoTime": "IO",
+    "java.lang.System::arrayCopy": "IO",
+    "java.lang.System::identityHashCode": "IO",
+    "java.lang.System::getProperties": "Env, Sys, IO",
+    "java.lang.System::lineSeparator": "Env, IO",
+    "java.lang.System::setProperties": "Env, Sys, IO",
+    "java.lang.System::getProperty": "Env, Sys, IO",
+    "java.lang.System::setProperty": "Env, Sys, IO",
+    "java.lang.System::clearProperty": "Env, Sys, IO",
+    "java.lang.System::getenv": "Env, IO",
+    "java.lang.System::getLogger": "Env, IO",
```

Comment:
I'm not quite sure what the logger is used for or what mental model to use, so I'm unsure if `Env` is correct. Help! :)